File: rust/src/models/effective_entitlement_list_response.rs
// this file is @generated
use serde::{Deserialize, Serialize};

use super::effective_entitlement::EffectiveEntitlement;

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct EffectiveEntitlementListResponse {
    pub data: Vec<EffectiveEntitlement>,
}

impl EffectiveEntitlementListResponse {
    pub fn new(data: Vec<EffectiveEntitlement>) -> Self {
        Self { data }
    }
}

File: python/meteroid/models/effective_entitlement_list_response.py
# this file is @generated
import dataclasses
import typing as t

from ..serialization import BaseModel
from .effective_entitlement import EffectiveEntitlement


@dataclasses.dataclass
class EffectiveEntitlementListResponse(BaseModel):
    data: t.List[EffectiveEntitlement]
